platformio.ini

```diff
@@ -12,4 +12,4 @@
 platform = ststm32
 framework = stm32cube
 board = nucleo_f401re
-build_flags = -DF4
\ No newline at end of file
+build_flags = -DF4 -I lib/stm32/STM32F4xx_HAL/Inc -I lib\stm32\CMSIS\Device\ST\STM32F4xx\Include -I lib\stm32\CMSIS\Include
```

Comment:
why are the adress using both` / `and `\ `?

Reply:
ugh...got catch. This explains an issue i had earlier. I'll amend.

Reply:
fixed in latest commit.

Reply:
That explains it!